doc/api/stream.md

```diff
@@ -418,14 +418,16 @@ changes:
   `Uint8Array`. For object mode streams, `chunk` may be any JavaScript value
   other than `null`.
 * `encoding` {string} The encoding if `chunk` is a string
-* `callback` {Function} Optional callback for when the stream is finished
+* `callback` {Function} Optional callback for when the stream finishes
+  or errors
 * Returns: {this}
 
 Calling the `writable.end()` method signals that no more data will be written
 to the [`Writable`][]. The optional `chunk` and `encoding` arguments allow one
 final additional chunk of data to be written immediately before closing the
 stream. If provided, the optional `callback` function is attached as a listener
-for the [`'finish'`][] event.
+for the [`'finish'`][] and the `'error'` event. In the latter case, an error
```

Comment:
This isn't technically true, the `callback` is not attached as an `'error'` event handler like `callback` is for `'finish'`, it's just called whether there's an error or not.

Reply:
https://github.com/nodejs/node/blob/ba684805b6c0eded76e5cd89ee00328ac7a59365/lib/_stream_writable.js#L708-L725

I think the `callback` isn't attached for either of the two. It's called in both cases.